jackson-apt-processor/src/main/java/org/dominokit/jacksonapt/processor/registration/JSONRegistrationProcessor.java

```diff
@@ -106,11 +115,10 @@ private MethodSpec createGetMethod(String name, String mapName, Class<?> returnT
 
     private FieldSpec createConstantMap(String name, Class<?> jsonType) {
         ClassName mapType = ClassName.get(Map.class);
-        ParameterizedTypeName classOfWildcard = ParameterizedTypeName.get(ClassName.get(Class.class),
-                WildcardTypeName.subtypeOf(Object.class));
+        ClassName typeClassName = ClassName.get(Type.class);
```

Comment:
given the ClassName.get("org.dominokit..." reference above, pick a convention on how you reference other types? generally prefer the string version to avoid classpath issues when the user runs the codegen, but in this case the jars might only work tightly coupled together, i'm not sure.

Reply:
Thanks, I will change it to string version to avoid possible conflicts.